cmake/openthread_extensions.cmake

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+set(NRFXLIB_DIR $ENV{ZEPHYR_BASE}/../nrfxlib)
```

Comment:
Don't expect `$ENV{ZEPHYR_BASE}`, that is not required to be set.

When referring between module, then `ZEPHYR_<module>_MODULE_DIR` shall be used, in this case, that would be `ZEPHYR_NRFXLIB_MODULE_DIR`.

But in this case, I actually believe this should not be needed in this case, see other comments.